### PR TITLE
fix(button): change active behaviour from box-shadow to background-color

### PR DESCRIPTION
**Pull request by valentin-mladenov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: CDE-871

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

BREAKING CHANGE: Change the default active state behavior of buttons to use background color instead of box-shadow on top.

### CODE REVIEW

File: projects/angular/src/button/STYLES.md [hunk 1]
```diff
@@ -2,159 +2,159 @@
...
+| --clr-btn-border-radius                         | Button border radius                                                       |
+| --clr-btn-border-width                          | Button border width                                                        |
+| --clr-btn-outline-bg-color                      | Default outline button background color                                    |
+| --clr-btn-default-color                         | Default button text color                                                  |
+| --clr-btn-default-bg-color                      | Default button background color                                            |
+| --clr-btn-default-border-color                  | Default button border color                                                |
+| --clr-btn-default-hover-color                   | Default button `:hover` text color                                         |
+| --clr-btn-default-hover-bg-color                | Default button `:hover` background color                                   |
+| --clr-btn-default-box-shadow-color              | Default button `:active` background color                                  |
```
Comment (dtsanevmw) [inline projects/angular/src/button/STYLES.md:23]:
```suggestion
| --clr-btn-default-active-bg-color              | Default button `:active` background color                                  |
```

**Review by dtsanevmw — CHANGES_REQUESTED** [review]
It looks okey for the core theme but for NG it seems that before the box-shadow was acting like a doubled border on top and now it's making it with a background which doesn't look right. 
Screenshot 2023-08-29 at 9 29 08
Screenshot 2023-08-29 at 9 28 48

File: projects/angular/src/button/STYLES.md (hunk 1 above)
Comment (valentin-mladenov) [reply projects/angular/src/button/STYLES.md]: corrected in commit 9040d78

**Comment by valentin-mladenov** [comment]
@dtsanevmw check my new changes regarding both `light` and `dark` NG themes. I tried with minimal changes to meet the requirements.

**Review by dtsanevmw — APPROVED** [review]
Looks good!

**Pull request merged**